### DISCUSSION TITLE
How do I run the setup commands?

### DISCUSSION THREAD

**Discussion by Userbingd** [AbdBarho/stable-diffusion-webui-docker#285] [Q&A]
The setup instruction are a little less than ideal in how they don't actually tell you how to run the commands. I tried using GIT, but that didn't work.

**Comment by AbdBarho** [comment]
```sh
git clone https://github.com/AbdBarho/stable-diffusion-webui-docker.git
cd stable-diffusion-webui-docker
docker compose --profile download up --build
docker compose --profile [ui] up --build
```
More info [here](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Setup)

**Comment by Userbingd** [comment]
I get bash: docker: command not found when I try to use it

**Reply by AbdBarho** [reply]
did you install docker?

**Reply by Userbingd** [reply]
I cloned the repo, then ran the docker compose --profile download up --build command, the setup instructions don't say to install anything other than to clone the repo

**Reply by AbdBarho** [reply]
I am pretty sure the [instructions](https://github.com/AbdBarho/stable-diffusion-webui-docker/wiki/Setup) say you need to have docker

![image](https://user-images.githubusercontent.com/24505302/211141608-57aff417-3dc8-47ba-ab4c-2083b7d3965c.png)

**Reply by niaodian** [reply]
if you can read Chinese, I would suggest to use this command to install docker and docker compose in just one-click:

bash <(curl -sSL https://cdn.jsdelivr.net/gh/SuperManito/LinuxMirrors@main/DockerInstallation.sh)
